spark_log_parser/__init__.py

```diff
@@ -1,2 +1,2 @@
 """Tools for providing Spark event log"""
-__version__ = "0.1.7"
+__version__ = "0.2.0"
```

Comment:
Since we are adding new fields to the output, bumping minor version seemed appropriate to me